HTML/js/rejestracja.js

```diff
@@ -0,0 +1,149 @@
+window.onload = function(e){
+    console.log('onload')
+    }
+    
+    function validatereg() {
+
+        let errors = true;
+        console.log(errors)
+        if (!emailvalidation('email')) {
+            errors = false;
+        }
+        if (!requiredvalidation('lastname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('firstname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('username')) {
+            errors = false;
+        }
+
+        if(!birthdatevalidation('birthdate')) {
+            errors = false;
+        }
+
+        if (!checkpassword('password')) {
+            errors = false;
+        }
+
+        if (!errors ){
+            document.getElementById("error").setAttribute("style", "display: inline;")
+        } else {
+            document.getElementById("error").setAttribute("style", "display: none;")
+        }
+    }
+
+    function emailvalidation(element){
```

Comment:
Expected indentation of 0 spaces but found 4.
Missing JSDoc comment.
Missing space before opening brace.